test/gitea_params_test.go

```diff
@@ -185,11 +185,11 @@ func TestGiteaParamsOnRepoCR(t *testing.T) {
 				"I am the most Kawaī params\nSHHHHHHH\nFollow me on my ig #nofilter\n{{ no_match }}\nHey I show up from a payload match\n{{ secret_nothere }}"), 2))
 }
 
-// TestParamshBodyHeadersCEL Test that we can access the body and headers in params as a CEL expression
-func TestParamshBodyHeadersCEL(t *testing.T) {
+// TestGiteaParamsBodyHeadersCEL Test that we can access the body and headers in params as a CEL expression and cel filter
+func TestGiteaParamsBodyHeadersCEL(t *testing.T) {
 	topts := &tgitea.TestOpts{
 		Regexp:      successRegexp,
-		TargetEvent: "pull_request, push",
```

Comment:
can we add another test for verifying `push` event for CEL expression

Reply:
good idea, i reworked the whole test including plenty of comments to explain how it does it... (let's do this in the future making sure all tests scenarios are fully commented)